lib/collection/src/shard/remote_shard.rs

```diff
@@ -39,11 +39,9 @@ pub struct RemoteShard {
     pub(crate) collection_id: CollectionId,
     pub peer_id: PeerId,
     ip_to_address: Arc<std::sync::RwLock<HashMap<u64, Uri>>>,
+    p2p_grpc_timeout: Duration,
```

Comment:
I kinda cheated here as the `RemoteShard` are not yet instantiated.
The configuration will have to be injected later on.